src/components/pages/Homepage.jsx

```diff
@@ -106,59 +113,90 @@ export default function Homepage({ homepage }) {
+                A Video Gallery of Poets
+                <br />
+                in Southern California
+              </h1>
+            </div>
+            <div className="featured" ref={featuredRef}>
+              <VideoPlayer video={thumbnailsList[activeThumbnail]} />
+            </div>
+          </div>
+          <div className="thumbnails">
+            <h3>Latest</h3>
+            <div className="carousel">
+              <ul className="carousel-track" ref={carouselRef}>
+                {homepage.youtubeVideoCollection.items.map((video, index) => (
+                  <li
+                    className={`carousel-item ${
+                      index === activeThumbnail ? "active" : ""
+                    }`}
+                    style={{ "--w": `${generatedThumbnailWidths[index]}px` }}
+                    key={index}
+                    ref={(el) => (thumbnailsRef.current[index] = el)}
+                  >
+                    <button onClick={() => handleThumbnailClick(index)}>
+                      <img src={video.thumbnail.url} alt={video.title} />
+                    </button>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          </div>
+        </main>
+      </>
+    );
+  }
 
-      carouselRef.current.scrollTo({ left: targetOffset, behavior: 'smooth' });
-      setActiveThumbnail(index);
-    },
-    []
-  );
-  return (<>
+  return (
     <main className="home page subgrid">
       <div className="top subgrid">
         <div className="intro">
-          <h1>A Video Gallery of Poets in Southern California</h1>
-          <Link href="/about">About Us</Link>
-        </div>
-        <div className="featured" ref={featuredRef}>
-          <VideoPlayer
-            video={thumbnailsList[activeThumbnail]}
-          />
+          <h1 className="body-text">
+            A Video Gallery of Poets
+            <br />
+            in Southern California
+          </h1>
         </div>
       </div>
       <div className="thumbnails">
-        <h3>Latest</h3>
         <div className="carousel">
           <ul className="carousel-track" ref={carouselRef}>
             {homepage.youtubeVideoCollection.items.map((video, index) => (
               <li
-                className={`carousel-item ${index === activeThumbnail ? "active" : ""
-                  }`}
-                style={{ "--w": `${generatedThumbnailWidths[index]}px` }}
+                className={`carousel-item ${
+                  index === activeThumbnail ? "active" : ""
+                }`}
                 key={index}
-                ref={el => thumbnailsRef.current[index] = el}
+                ref={(el) => (thumbnailsRef.current[index] = el)}
               >
-                <button
-                  onClick={() => handleThumbnailClick(index)}>
-                  <img src={video.thumbnail.url} alt={video.title} />
+                <button onClick={() => handleThumbnailClick(index)}>
```

Comment:
Since these buttons contain LOTS of text, their accessible name gets really long. An aria-label would make them more digestible. Maybe something like `Focus ${video.title}`